packages/lit-dev-content/site/docs/composition/component-composition.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Component composition
+eleventyNavigation:
+  parent: Composition
+  key: Component composition
+  order: 2
+---
+
+The most common way to handle complexity and factor Lit code into separate units is _component composition_: that is, the process of building a large, complex component out of smaller, simpler components. Imagine you've been tasked with implementing a screen of UI:
+
+![Screenshot of an application that displays a set of animal photos. The application has a top bar with a title ("Fuzzy") and a menu button. A left menu drawer is open, showing a set of options.](/images/docs/composition/fuzzy-screenshot.png)
+
+
+You can probably identify the areas which will involve some complexity to implement. Chances are, those could be components.
+
+By isolating the complexity into specific components, you make the job much simpler, and you can then compose these components together to create the overall design.
+
+For example, the fairly simple screenshot above involves a number of possible components: a top bar, a menu button,  a drawer with menu items for navigating the current section; and a main content area. Each of these could be represented by a component. A complex component, like a drawer with a navigation menu, might be broken into many smaller components: the drawer itself, a button to open and close the drawer, the menu, individual menu items.
+
+Lit lets you compose by adding elements to your template—whether those are built-in HTML elements or custom elements.
+
+```js
+render() {
+  return html`
+    <top-bar>
+      <icon-button icon="menu" slot="nav-button"></icon-button>
+      <span slot="title">Fuzzy</span>
+    </top-bar>
+    `;
+}
+```
+
+
+In addition to the nodes in your shadow DOM, you can render child nodes, provided by the component user. For example, the `top-bar` element in the previous example has two children supplied by the user: a navigation button, and a title.
```

Comment:
I looked at our current shadow DOM section and we don't currently use "light DOM" there.

I also looked at the DOM spec, and they now use the term "light tree"...  We could consider updating the language in the shadow DOM section.

Reply:
"light DOM" is very common colloquially at least, and it does draw immediate contrast to "shadow DOM" which is pretty useful.

Reply:
Yeah. I'm suggesting we should add light DOM/light tree to the current shadow DOM section. I can add it here, but it's kind of funky if it's not mentioned in the main shadow DOM section.

Reply:
I made this into its own section and now it feels kind of out of place. I'm wondering whether it should be moved to the end of the doc--everything else is talking about shadow DOM composition. This might make more sense as a section at the end describing how working with light DOM children is a different kind of composition and

Reply:
I think what's there is good for now, but maybe bookmark it as an area to expand on.